### ISSUE TITLE
fontLoader fails when using pod

### ISSUE THREAD

**Issue by mtissington** [thii/FontAwesome.swift#95]
Bundle.url(...)! generates an exception when otf is not found ... the following resolve the problem

```
private class FontLoader {
    class func loadFont(_ name: String) {
        let bundle = Bundle(for: FontLoader.self)
        let identifier = bundle.bundleIdentifier
        var fontURL: URL
        if identifier?.hasPrefix("org.cocoapods") == true {
            // If this framework is added using CocoaPods, resources is placed under a subdirectory
            fontURL = bundle.url(forResource: name, withExtension: "otf", subdirectory: "FontAwesome.swift.bundle")!
        } else {
           fontURL = bundle.url(forResource: name, withExtension: "otf")!
        }```

**Comment by mtissington** [comment]
It seems the 2.3 branch is working correctly, it's only broken in the 3.0 branch.

**Comment by PVKonovalov** [comment]
I have the save error for 0.11.1 tag with "fatal error: unexpectedly found nil while unwrapping an Optional value" message on that string:
var fontURL = bundle.url(forResource: name, withExtension: "otf")!
in FontAwesome.swift file. I think the swift program does not found the font in the bundle.

**Comment by PVKonovalov** [comment]
I think the problem is in this place of FontAwesome.swift file:
var fontURL = bundle.url(forResource: name, withExtension: "otf")!
For CocoaPods that method is return nil. If you remove wrapping of  that string and add wrap to  
let data = try! Data(contentsOf: fontURL!)
error will fixed.

**Comment by savagegus** [comment]
I had the same problem and this worked for me, +1

**Comment by thii** [comment]
Fixed in `1.0.0`.
